src/style/global.scss

```diff
@@ -33,6 +33,14 @@ html {
   &.iui-theme-dark-hc {
     @include theme-variables(dark-hc);
   }
+
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
```

Comment:
Why did it this way? Just adding for all to have `bordex-box` did not work out?

Reply:
I was following the[ recommendation I found at css-tricks](https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/).